SingularityUI/app/templates/vex/requestScale.hbs

```diff
@@ -1,2 +1,13 @@
 <p>Enter the desired number of instances to run for request:</p>
 <pre>{{ id }}</pre>
+
+<label for="bounce" id="bounce-label">
+    <input type="checkbox" id="bounce"{{#if bounceAfterScale}} checked{{/if}}>
+    Bounce after scaling
+</label>
```

Comment:
right now the incremental option isn't show if bounce checkmark is unchecked and appears when you check it. would you rather see it greyed out instead of having it hidden?